<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskmanager</class>
 <widget class="QMainWindow" name="taskmanager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>taskmanager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>526</width>
      <height>438</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListView" name="listView1">
           <property name="font">
            <font>
             <family>Segoe Print</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="selectionRectVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListView" name="listView2">
           <property name="font">
            <font>
             <family>Segoe Print</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>true</bool>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="dragDropMode">
            <enum>QAbstractItemView::DragDrop</enum>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::MoveAction</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listView3">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_OPtions">
    <property name="title">
     <string>&amp;OPtions</string>
    </property>
    <addaction name="actionDelete"/>
    <addaction name="actionShow_All"/>
    <addaction name="actionShow_Pending"/>
    <addaction name="actionShow_Finished"/>
    <addaction name="actionShow_Today_tasks"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_OPtions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionShow_Finished"/>
   <addaction name="actionShow_Pending"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="new_file" resource="icone2.qrc">
     <normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.01 (1).jpeg</normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.01 (1).jpeg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icone2.qrc">
     <normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.00.jpeg</normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.00.jpeg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionShow_Pending">
   <property name="icon">
    <iconset resource="icone2.qrc">
     <normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.01.jpeg</normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.01.jpeg</iconset>
   </property>
   <property name="text">
    <string>Show Pending tasks</string>
   </property>
  </action>
  <action name="actionShow_Finished">
   <property name="icon">
    <iconset resource="icone2.qrc">
     <normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.01 (2).jpeg</normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.01 (2).jpeg</iconset>
   </property>
   <property name="text">
    <string>Show Finished tasks</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionShow_Today_tasks">
   <property name="text">
    <string>Show Today tasks</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icone2.qrc">
     <normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.02.jpeg</normaloff>:/new/prefix1/icone2/WhatsApp Image 2022-01-21 at 21.52.02.jpeg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionShow_All">
   <property name="text">
    <string>Show All</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icone2.qrc"/>
 </resources>
 <connections/>
</ui>
